app/web/src/newhotness/logic_composables/attribute_tree.ts

```diff
@@ -114,13 +114,16 @@ export const makeSavePayload: MakePayload = (
   // TODO - Paul there's a better way to handle this for sure!
   let coercedVal: string | boolean | number | null = value;
 
+  const payload: componentTypes.UpdateComponentAttributesArgs = {};
```

Comment:
Comment for the future: since this is only being used in one place, may as well just move this to line 126 and do `return { [path]: { $source: null } };`